src/dynamics/video_player/player/player.js

```diff
@@ -810,8 +810,9 @@ Scoped.define("module:VideoPlayer.Dynamics.Player", [
                         if (height) styles.height = isNaN(height) ? height : parseFloat(height).toFixed(2) + "px";
                         if (width) styles.width = isNaN(width) ? width : parseFloat(width).toFixed(2) + "px";
                         containerStyles = floatingonly ? {} : Objs.extend({}, styles);
-                        if (!gallerySidebar && showsidebargallery && layout === "desktop" && !fullscreened)
+                        if (gallerySidebar && showsidebargallery && layout === "desktop" && !fullscreened) {
```

Comment:
Hi @Josewd, this causes layout shifting issues when the player goes to floating state. 😬 
Removing `gallerySidebar` from the condition fixes it.
```suggestion
                        if (showsidebargallery && layout === "desktop" && !fullscreened) {
```

Reply:
Done!

Reply:
Now aspect-ratio looks broken on outstream, looks like we need to add outstream condition here after all. @Josewd sorry for making you change this over and over, these sidebar gallery settings are confusing and the way we're setting aspect-ratio right now is really messy. 😬